.mage/version.go

```diff
@@ -87,6 +88,11 @@ func bumpVersion(bump string) error {
 	}
 	var newVersion semver.Version
 	switch bump {
+	case "release":
```

Comment:
It would be nicer to make these an enum IMO

Reply:
It wouldn't be an enum in Make either. Let's not over engineer the build tooling, please.